CustomApps/lyrics-plus/index.js

```diff
@@ -29,10 +29,14 @@ function getConfig(name, defaultVal = true) {
 const APP_NAME = "lyrics-plus";
 
 // Modes enum
-const KARAOKE = 0,
-	SYNCED = 1,
-	UNSYNCED = 2,
-	GENIUS = 3;
+const Modes = {
+	KARAOKE: 0,
+	SYNCED: 1,
+	UNSYNCED: 2,
+	GENIUS: 3
+};
+
+const getKeyByValue = (obj, value) => Object.keys(obj).find(key => obj[key] === value);
```

Comment:
Object only serves the purpose of the function below, which was only used once and is not needed. This also adds a lot of boilerplate code.
Revert all unnecessary changes following this.

Reply:
i will finish this up probably today